server/src/gateways/judges.gateway.ts

```diff
@@ -41,18 +42,19 @@ export class JudgesGateway implements FightEndConditionFulfilledObserver {
     @MessageBody('judgeId') judgeId: string,
     @ConnectedSocket() client: Socket,
   ) {
-    const status: Response = {
+    const response: JoinResponse = {
       status: this.fightsService.addJudge(fightId, judgeId, client),
+      redPlayer: null,
```

Comment:
Tutaj celowo był na początku sam `status`, a nie od razu `response`. Według stosowanej do tej pory konwencji, przy niepowodzeniu odsyłaliśmy prostu obiekt `{ status: STATUS_CODE }`, a tutaj odpowiedź przy błędzie to `{ status: STATUS_CODE, redPlayer: null, bluePlayer: null }`. Było to też zaznaczone w pliku z opisem endpointów "If there are any errors, endpoints return { status: STATUS_NAME } object by default"